### ISSUE TITLE
Allow renaming of printers in LAN Mode

### ISSUE THREAD

**Issue by SoCuul** [SoftFever/OrcaSlicer#3784]
**Is your feature request related to a problem? Please describe.**
Currently printers display an auto-generated name while connected via LAN mode. This makes it confusing to differentiate between printers.

**Which printers will be beneficial to this feature**
Bambu Lab X1/P1/A1

**Describe the solution you'd like**
It would be nice if we could set an alias in OrcaSlicer to display before (or replace) the auto generated name of LAN printers. The functionality could resemble the renaming of printers when connected via online mode.

**Comment by rspeed** [comment]
Bambu seems unwilling to implement this in their own software. Presumably because it would require a modification to the printer firmware. But a basic mapping on the client end would indeed serve as a suitable bandaid in the meantime.

**Comment by SoCuul** [comment]
@DanBao-Bambu is this planned to be implemented? printer-side would be difficult, so just being able to create an alias for the printer locally would be perfect.

Eg. 3DP-01P-123 would appear instead as "My Printer" after changing a local configuration setting. It would show this alias name in all references to the printer's name, just like printers within bambu cloud.

**Comment by nate1mace** [comment]
Is there still no solution to this? Seems like Orca-side mapping would quick thing to implement.

**Comment by SoCuul** [comment]
@SoftFever Is this not planned for OrcaSlicer at all, or should a new issue be created?

**Comment by SoCuul** [comment]
@nate1mace after checking in the latest version of OrcaSlicer. It seems to correctly display the printer's name (after being configured while connected to the cloud.

I renamed my printer, then turned on lan mode and it shows the name just fine now!

<img width="207" alt="Image" src="https://github.com/user-attachments/assets/285ccb76-268d-4ecb-b62f-dbaf210ecbbf" />

Not perfect, but definitely more than workable.

**Comment by f0x52** [comment]
It's a bit tricky, but I also found the [bambu-ssdp-discovery.py](https://gist.github.com/Alex-Schaefer/72a9e2491a42da2ef99fb87601955cc3) to work for setting a custom name

**Comment by nate1mace** [comment]
It seemed to me that the very first configuration of the name in an earlier firmware was what my P1S was getting stuck on. I tried this route - which seemed to mostly fix the issue. Also a bit tricky, and only successful for 3 of 4 P1S devices that I tried

1 - Turn off LAN and bind with handy app
2 - In app settings, depreciate firmware to 1.05.01.00
3 - In 1.05, factory reset the P1S from LCD
4 - Scan QR with handy app and confirm with new name you want to change
5 - In app settings, bring firmware back to 1.07 (highest firmware which still allows LAN use)
6 - On printer LCD, it should now have the name you configured while in 1.05 firmware
7 - On Handy app, press delete device to unbind. On printer, turn back on LAN mode.
8 - On Orca perform a fresh rebind by pressing +Bind with access code (don’t select the printer in drop down)

Just step 8 gave me trouble, one of four printers didn't update on Orca, despite the LCD on the printer confirming it was renamed. This again would likely be easiest with a simple remapping on the Orca side.
